### DISCUSSION TITLE
Printout 1 day instrument file to 24x1 hour plots

### DISCUSSION THREAD

**Discussion by Bemtgenj** [groops-devs/groops#237] [Q&A]
Hello,

Is there a possibility to print for example 24 hourly plots out of 1 day concatenated instrument file? I have a file with data that covers an entire day but there is too many data points to see a proper plot. So i thought of printing it out in 24 x 1hour periods or 30 min periods if there is still too many data. I have managed to get hourly files set up but the plot always shows the same first data (1st hour) for all 24 plots (see script print screen below). Is there a way to define  a variable with 1 hour increments? Or is there a program that does the inverse of concatenation?

The other idea was to set up a series with  24 x1 hour files. here too i ran into the same problem as above. I could not get the variable working that does 1 hour loops. I got 24 files but the fata itself in the files was not 1hour periods
![Screenshot from 2024-02-14 13-58-47](https://github.com/groops-devs/groops/assets/100558965/7c9f6767-4347-4244-9eee-944f57796d55)
   
![Screenshot from 2024-02-14 14-11-40](https://github.com/groops-devs/groops/assets/100558965/310efb29-71a2-47f2-ac19-c273632886c8)

Thank you very much for your feedback and help
Cheers
Jean

**Comment by tmayerguerr** [comment]
Hi Jean,

Two possible ways:

1.  Restrict the time interval in [PlotGraph](https://groops-devs.github.io/groops/html/PlotGraph.html)->[axisX](https://groops-devs.github.io/groops/html/plotAxisType.html)(Time) with (min, max).
2. Or divide your instrument file into smaller segments before by cutting out [timeIntervals](https://groops-devs.github.io/groops/html/timeSeriesType.html) with [InstrumentSynchronize](https://groops-devs.github.io/groops/html/InstrumentSynchronize.html).

Cheers
Torsten

**Comment by Bemtgenj** [comment]
Hi Torsten

Txs for the quick feedback. I will try your 2 possible ways and get back to you if i have further questions / problems

Cheers
Jean
